src/SprykerEco/Zed/Econda/Business/Exporter/Runner.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Econda\Business\Exporter;
+
+use Generated\Shared\Transfer\BatchResultTransfer;
+use Generated\Shared\Transfer\LocaleTransfer;
+use Spryker\Shared\Kernel\Store;
+use SprykerEco\Zed\Econda\Business\Exporter\Exception\BatchResultException;
+use SprykerEco\Zed\Econda\Dependency\Facade\EcondaToLocaleInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class Runner implements RunnerInterface
+{
+    /**
+     * @var \SprykerEco\Zed\Econda\Business\Exporter\ExporterInterface
+     */
+    protected $exporter;
+
+    /**
+     * @var \SprykerEco\Zed\Econda\Dependency\Facade\EcondaToLocaleInterface
+     */
+    protected $localeFacade;
+
+    /**
+     * @param \SprykerEco\Zed\Econda\Dependency\Facade\EcondaToLocaleInterface $localeFacade
+     * @param \SprykerEco\Zed\Econda\Business\Exporter\ExporterInterface $exporter
+     */
+    public function __construct(
+        EcondaToLocaleInterface $localeFacade,
+        ExporterInterface $exporter
+    ) {
+        $this->localeFacade = $localeFacade;
+        $this->exporter = $exporter;
+    }
+
+    /**
+     * @param \Symfony\Component\Console\Output\OutputInterface $output
+     *
+     * @return array
+     */
+    public function runExport(OutputInterface $output)
+    {
+        $storeCollection = Store::getInstance()->getAllowedStores();
+
+        $results = [];
+
+        foreach ($storeCollection as $storeName) {
+            $output->writeln('');
+            $output->writeln('<fg=yellow>----------------------------------------</fg=yellow>');
+            $output->writeln(sprintf(
+                '<fg=yellow>Exporting Store:</fg=yellow> <fg=white>%s</fg=white>',
+                $storeName
+            ));
+            $output->writeln('');
+
+            $localeCollection = Store::getInstance()->getLocalesPerStore($storeName);
```

Comment:
Same here — direct dependency is not welcome

Reply:
Here, you will have trouble because StoreFacade never had locale specific questions. This actually opens up a question, why do you need it now. What does this exporter do? Why does it try to export data from other store?